lydia-braga/1.iii/livro.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
As this document is in portuguese language we need to update the attribute `lang` to "pt-br"